apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js

```diff
@@ -123,10 +123,6 @@ describe('The JSDebugger component', () => {
       expect(openIcon()).to.exist;
     });
 
-    it('will collapse the debugger by setting the height in the css', () => {
```

Comment:
I removed this test because it was relying on a constant within a css file, which was no longer working in tests with css modules, and it didn't seem like the scope of a unit test to verify a specific css constant.